src/engraving/dom/beambase.h

```diff
@@ -34,6 +34,40 @@ enum class BeamType {
     TREMOLO
 };
 
+//---------------------------------------------------------
+//   BeamFragment
+//    position of primary beam
+//    idx 0 - DirectionV::AUTO or DirectionV::DOWN
+//        1 - DirectionV::UP
```

Comment:
Aside: I've always found "auto/down" vs "up" a strange classification and never understood why we'd ever treat "auto" as "down". But that's probably for another day.